CHANGELOG.md

```diff
@@ -86,6 +86,8 @@
   * Card
     * Remove `threeDSecureInfo` from `CardNonce`
     * Move `ThreeDSecureInfo` to `three-d-secure` module
+    * Add `CardResult` object
+    * Change `CardTokenizeCallback` parameters
```

Comment:
Since we are pretty much changing the parameters of every public method and callback throughout the SDK for this version, I've just been saying "change parameters" to keep the CHANGELOG less verbose. The detailed changes are in the migration guide though. So I don't know because yes it might be helpful to see the actual parameter changes, but also for every method that is changing the CHANGELOG will get very long and verbose. What do you think?

Reply:
Yeah I think we were pretty verbose in the iOS changelog, it's quite long for the betas. No idea if it's helpful or not though.